src/main/java/org/threeten/extra/chrono/InternationalFixedChronology.java

```diff
@@ -0,0 +1,490 @@
+    /**
+     * Serialization version UID.
+     */
+    private static final long serialVersionUID = -8252657100538813526L;
+    /**
+     * Standard 7-day week.
+     */
+    static final int DAYS_IN_WEEK = 7;
+    /**
+     * In all months, there are 4 complete weeks.
+     */
+    static final int WEEKS_IN_MONTH = 4;
+    /**
+     * There are 13 months in a year.
+     */
+    static final int MONTHS_IN_YEAR = 13;
+    /**
+     * There are 4 weeks of 7 days, or 28 total days in a month.
+     */
+    static final int DAYS_IN_MONTH = WEEKS_IN_MONTH * DAYS_IN_WEEK;
+    /**
+     * Range of aligned day-of-week.
+     */
+    static final ValueRange ALIGNED_DAY_OF_WEEK_RANGE = ValueRange.of(0, DAYS_IN_WEEK);
+    /**
+     * Range of day-of-week.
+     */
+    static final ValueRange DAY_OF_WEEK_RANGE = ValueRange.of(0, 1, 0, DAYS_IN_WEEK);
+    /**
+     * There are 13 months of 28 days, or 365 days in a (non-leap) year.
+     */
+    static final int DAYS_IN_YEAR = MONTHS_IN_YEAR * DAYS_IN_MONTH + 1;
+    /**
+     * There are 52 weeks in a year.
+     */
+    static final int WEEKS_IN_YEAR = DAYS_IN_YEAR / DAYS_IN_WEEK;
+    /**
+     * The number of days in a 400 year cycle.
+     */
+    static final int DAYS_PER_CYCLE = 146097;
+    /**
+     * The number of days from year zero to year 1970, still the era only allows year 1 and higher.
+     * There are five 400 year cycles from year zero to 2000.
+     * There are 7 leap years from 1970 to 2000.
+     */
+    static final long DAYS_0000_TO_1970 = (DAYS_PER_CYCLE * 5L) - (30L * 365L + 7L);
+    /**
+     * Range of year.
+     */
+    static final ValueRange YEAR_RANGE = ValueRange.of(1, 1_000_000L);
+    /**
+     * Epoch day range.
+     */
+    static final ValueRange EPOCH_DAY_RANGE = ValueRange.of(-DAYS_0000_TO_1970, 1_000_000L * DAYS_IN_YEAR + getLeapYearsBefore(1_000_000L) - DAYS_0000_TO_1970);
+    /**
+     * Range of proleptic month.
+     */
+    private static final ValueRange PROLEPTIC_MONTH_RANGE = ValueRange.of(13, 1_000_000 * 13L - 1);
+    /**
+     * Range of day of month.
+     */
+    static final ValueRange DAY_OF_MONTH_RANGE = ValueRange.of(-1, 0, -1, DAYS_IN_MONTH);
+    /**
+     * Range of week of year.
+     */
+    static final ValueRange WEEK_OF_YEAR_RANGE = ValueRange.of(0, WEEKS_IN_YEAR);
+    /**
+     * Range of aligned week of month, the weeks are always perfectly aligned.
+     */
+    static final ValueRange WEEK_OF_MONTH_RANGE = ValueRange.of(0, 1, 0, WEEKS_IN_MONTH);
+    /**
+     * Range of day of year.
+     */
+    static final ValueRange DAY_OF_YEAR_NORMAL_RANGE = ValueRange.of(1, DAYS_IN_YEAR);
+    /**
+     * Range of day of leap year.
+     */
+    static final ValueRange DAY_OF_YEAR_LEAP_RANGE = ValueRange.of(1, DAYS_IN_YEAR + 1);
+    /**
+     * Range of day of year, inclusive
+     */
+    static final ValueRange DAY_OF_YEAR_RANGE = ValueRange.of(1, DAYS_IN_YEAR, DAYS_IN_YEAR + 1);
```

Comment:
you don't need to supply a custom value for `DAY_OF_YEAR_RANGE`, given it matches ISO.